### PR TITLE
Feat: monster importer stat block formatting 

### PR DESCRIPTION
**Pull request by arcos**
This just makes the formatting for the monster importer more closely match the monsters that come from the compendium

### CODE REVIEW

**Comment by PrototypeESBU** [comment]
@arcos This is a nice touch, I like it. There's just some problems with your regex that don't account for special characters in features titles. See lich for example:
image
or
image

**Comment by arcos** [comment]
@PrototypeESBU  "Fixed". Though I am wondering if we should include other characters. I don't seem to see anything different than this pattern in the stat blocks.

**Comment by PrototypeESBU** [comment]
Best to not make any assumptions. I think in parsing these I just matched everything that wasn't a "." leading up to the first "."

File: data/packs/documentation.db/release_notes__CG5T5H4KazrCJ9nt.json
Comment (Muttley) [inline data/packs/documentation.db/release_notes__CG5T5H4KazrCJ9nt.json:1]: Please don't check in this file, it is auto-generated at build time.

Comment (arcos) [reply data/packs/documentation.db/release_notes__CG5T5H4KazrCJ9nt.json:1]: @Muttley Should this be gitignored then?

**Comment by PrototypeESBU** [comment]
I rolled up these changes as part of #698

**Pull request closed without merging**